src/Components/BreathingRoom/BreathingRoomAnimation.js

```diff
@@ -1,59 +1,69 @@
 import gsap from 'gsap'
 
-export const inhaleExhale = () => {
+export const inhaleExhale = (mood) => {
+  let colors = {
+    fine: ['hsl(128, 29%, 90%)', 'hsl(200, 14%, 96%)', 'hsl(119, 38%, 72%)',
+      'hsl(120, 18%, 56%)', 'hsl(124, 11%, 24%)', 'hsl(0, 0%, 8%)'],
+    anxious: ['hsl(49, 43%, 71%)', 'hsl(200, 14%, 96%)', 'hsl(48, 46%, 51%)',
+      'hsl(48, 57%, 45%)', 'hsl(48, 44%, 25%)', 'hsl(225, 10%, 8%)'],
+    angry: ['hsl(325, 15%, 78%)', 'hsl(200, 14%, 96%)', 'hsl(327, 18%, 52%)',
+        'hsl(329, 57%, 25%)', '	hsl(327, 52%, 21%)', 'hsl(320, 30%, 12%)'],
```

Comment:
There appears to be a giant space in one of these strings. Is this a GitHub problem or a spacing issue we need to address?

Reply:
I'll take a look on my syntax sweep later today :)